mantidproject/mantid
# Functions default values

Fixes #13346.

**To test**

Try fitting peaks with Gaussian, Lorentzian or BackToBackExponential without setting any initial values. Fit algorithm tries to find suitable values itself. BackToBackExponential may still fail the fit. In this case run Fit with MaxIterations=0 and check that the parameters set to the function are reasonable.
